### PR TITLE
Fix serial been occupied cause test failed issue

### PR DESCRIPTION
**Pull request by kiya956**
impact:
    sanity/agent/console.py

description:
    add fuser to kill the process who occupy serial

test:
    N/A

### CODE REVIEW

File: sanity/agent/console.py [hunk 1]
```diff
@@ -30,6 +30,7 @@ def __init__(self, uname, com_port="/dev/ttyUSB0", brate=115200):
                 break
             except serial.SerialException as e:
                 print("{} retrying.....".format(e))
+                syscmd("fuser -k " + com_port)
```
Comment (tsunghanliu) [inline sanity/agent/console.py:33]: IMO, we should verify if the serial port exists first before open it.
Otherwise, it will run in this loop forever.

**Review by tsunghanliu — CHANGES_REQUESTED** [review]
I don't feel it's good enough to handle the exception in this way.

File: sanity/agent/console.py (hunk 1 above)
Comment (kiya956) [reply sanity/agent/console.py:40]: Hi Robert, I add the check for device, if device not exist, we just terminate the test.

**Review by tsunghanliu — APPROVED** [review]
LGTM

**Pull request merged**